apps/web/src/components/modals/alertModal.tsx

```diff
@@ -1,48 +1,77 @@
-import { useState, type FunctionComponent } from 'react';
+import { type FunctionComponent, useState } from "react";
+
 import {
-    AlertDialog,
-    AlertDialogAction,
-    AlertDialogCancel,
-    AlertDialogContent,
-    AlertDialogDescription,
-    AlertDialogFooter,
-    AlertDialogHeader,
-    AlertDialogTitle,
-    AlertDialogTrigger,
-} from "~/components/ui/alert-dialog"
-import { Input } from '../ui/input';
+  AlertDialog,
+  AlertDialogAction,
+  AlertDialogCancel,
+  AlertDialogContent,
+  AlertDialogDescription,
+  AlertDialogFooter,
+  AlertDialogHeader,
+  AlertDialogTitle,
+  AlertDialogTrigger,
+} from "~/components/ui/alert-dialog";
+
+import { Input } from "../ui/input";
+
 type AlertModalProps = {
-    isOpen: boolean;
-    onOpenChange: (open: boolean) => void;
-    title: string;
-    description: string;
-    unlockString?: string;
-    onCancel: () => void;
-    onConfirm: () => void;
+  isOpen: boolean;
+  onOpenChange: (open: boolean) => void;
+  title: string;
+  description: string;
+  unlockString?: string;
+  onCancel: () => void;
+  onConfirm: () => void;
 };
 
-export const AlertModal: FunctionComponent<AlertModalProps> = ({ isOpen, onOpenChange, title, description, unlockString, onCancel, onConfirm, }) => {
-    const [inputValue, setInputValue] = useState('');
+export const AlertModal: FunctionComponent<AlertModalProps> = ({
+  isOpen,
+  onOpenChange,
+  title,
+  description,
+  unlockString,
+  onCancel,
+  onConfirm,
+}) => {
```

Comment:
```suggestion
export function AlertModal({
  isOpen,
  onOpenChange,
  title,
  description,
  unlockString,
  onCancel,
  onConfirm,
}): AlertModalProps {
```

Reply:
}): AlertModalProps should be }: AlertModalProps )  but still thanks for the help